packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FLTGoogleMapJSONConversions.m

```diff
@@ -78,7 +78,7 @@ + (nullable NSDictionary *)dictionaryFromCoordinateBounds:(GMSCoordinateBounds *
   };
 }
 
-+ (nullable GMSCameraPosition *)cameraPostionFromDictionary:(nullable NSDictionary *)data {
++ (nullable GMSCameraPosition *)cameraPositionFromDictionary:(nullable NSDictionary *)data {
```

Comment:
Our policy has generally been (except when there's a very specific reason to believe otherwise, which has been super rare) to assume that nobody interacts directly with the native plugin code, so we don't treat changes to its interfaces as breaking. Instead we treat it as an implementation detail of the package.